src/libaktualizr/package_manager/ostreemanager.cc

```diff
@@ -292,7 +292,7 @@ Json::Value OstreeManager::getInstalledPackages() const {
 
 Uptane::Target OstreeManager::getCurrent() const {
   GObjectUniquePtr<OstreeSysroot> sysroot_smart = OstreeManager::LoadSysroot(config.sysroot);
-  OstreeDeployment *booted_deployment = ostree_sysroot_get_booted_deployment(sysroot_smart.get());
+  g_autoptr(OstreeDeployment) booted_deployment = ostree_sysroot_get_booted_deployment(sysroot_smart.get());
```

Comment:
I think it will cause an issue in case of a real ostree, not mocked one (`ostree-sysroot.c`). I am gonna fix the memory leak in `aktualizr/tests/ostree_mock.c` anyway

Reply:
I studied the ostree source code with more details and can confirm that ostree expects that OstreeDeployment instance is owned by OstreeSysroot instance. Any client calling ostree_sysroot_get_booted_deployment() should get a pointer to the OstreeDeployment without any ownership transfer and should not do any reference decrement and/or the OstreeDeployment instance deletion, this is a task of the entity that created this instance. So, I am gonna fix the issue here https://github.com/advancedtelematic/aktualizr/blob/ccf304a93df743beb801699401aafdc1122d4cf7/tests/ostree_mock.c#L133

Reply:
ah - i need to do a similar fix here also: https://github.com/advancedtelematic/aktualizr/blob/master/src/aktualizr_lite/ostree_mock.cc

I'll fix that so that this PR passes valgrind

Reply:
I think i've finally got this working cleanly: https://github.com/advancedtelematic/aktualizr/commit/3a01b3a95aaf7b1548aa30afcd52f195ea25a171